### ISSUE TITLE
[Feedback] [Accessibility] Visual Studio Code for the Web

### ISSUE THREAD

**Issue by MasterInQuestion** [microsoft/vscode#166103]
Touch screen unfriendly.
    Entirely icon-based UI lacking adequate text explanation highly confusing (in particular when the custom fonts failed to load).
    .
    Typically would have to refer to Visual Studio Code's documentations to use. (while there lacks adequate linking to which)
    Even reading the documentations wouldn't be able to resolve certain situations: e.g. the "Search".

&#x2D;---

Version: 1.73.1
Commit: 6261075646f055b99068d3688932416f2346dd3b
User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:106.0) Gecko/20100101 Firefox/106.0
Embedder: github.dev

**Comment by MasterInQuestion** [comment]
[[
=== VSCode Web ===

    [
    |&#x2A;| Title: The "github.dev" web-based editor
    |&#x2A;| URL: [https://docs.github.com/en/codespaces/the-githubdev-web-based-editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor) ]

    [
    |&#x2A;| Title: Visual Studio Code for the Web
    |&#x2A;| URL: [https://code.visualstudio.com/docs/editor/vscode-web](https://code.visualstudio.com/docs/editor/vscode-web) ]

    [
    |&#x2A;| Title: "vscode.dev" Announcement
    |&#x2A;| URL: [https://code.visualstudio.com/blogs/2021/10/20/vscode-dev](https://code.visualstudio.com/blogs/2021/10/20/vscode-dev) ]

    [
    |&#x2A;| Title: User Interface - Visual Studio Code
    |&#x2A;| URL: [https://code.visualstudio.com/docs/getstarted/userinterface](https://code.visualstudio.com/docs/getstarted/userinterface) ]

    [
    |&#x2A;| Title: Basic Editing - Visual Studio Code # Find In Selection
    |&#x2A;| URL: [https://code.visualstudio.com/docs/editor/codebasics#_find-in-selection](https://code.visualstudio.com/docs/editor/codebasics#_find-in-selection) ]

    [
    |&#x2A;| Title: Code Navigation - Visual Studio Code
    |&#x2A;| URL: [https://code.visualstudio.com/docs/editor/editingevolved](https://code.visualstudio.com/docs/editor/editingevolved) ]

    [
    |&#x2A;| Title: Accessibility - Visual Studio Code
    |&#x2A;| URL: [https://code.visualstudio.com/docs/editor/accessibility](https://code.visualstudio.com/docs/editor/accessibility) ]

    [
    |&#x2A;| Title: Source Control with Git - Visual Studio Code # Git Status Bar actions
    |&#x2A;| URL: [https://code.visualstudio.com/docs/sourcecontrol/overview#_git-status-bar-actions](https://code.visualstudio.com/docs/sourcecontrol/overview#_git-status-bar-actions) ]
]]

**Comment by isidorn** [comment]
Can you clarify and provide reproducable steps?
What do you mean touch screen unfriendly? It works just fine with my touch screen.

**Comment by MasterInQuestion** [comment]
Try "Search".
    What does each of the buttons mean?

    The problem is in particular serious when the browser does not properly handle "mouseover" alike.

**Comment by MasterInQuestion** [comment]
May be worth of reference:
    [ Quote (on iconized UI representation):
[https://forum.vivaldi.net/post/596774](https://forum.vivaldi.net/post/596774)
    The iconized UI should consider adding tiny text below the graphic for better clarity.
    Notable examples:
    ...
<.>    Which all used some not too straight forward symbols that would need clarification.
    .
    The text (within 12 character-width) may be completely fit into the area. Using some cryptic acronyms would probably not serve the purpose. ]

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!

**Comment by MasterInQuestion** [comment]
By not interfering with the zoom gestures, and anyhow applying the iconized UI representation suggestion:
    The accessibility issues should be over-half solved.

    As for the RegEx documentations, I believe it's .NET style RegEx?
    Just adding links to which should suffice.
    .
    Filename globs, dunno.
